Nodejs/Product/Nodejs/Commands/DiagnosticsForm.Designer.cs

```diff
@@ -50,12 +51,12 @@ partial class DiagnosticsForm {
             this._ok.AutoSize = true;
             this._ok.AutoSizeMode = System.Windows.Forms.AutoSizeMode.GrowAndShrink;
             this._ok.DialogResult = System.Windows.Forms.DialogResult.Cancel;
-            this._ok.Location = new System.Drawing.Point(732, 473);
-            this._ok.Margin = new System.Windows.Forms.Padding(6, 3, 6, 3);
-            this._ok.MinimumSize = new System.Drawing.Size(86, 24);
+            this._ok.Location = new System.Drawing.Point(976, 581);
+            this._ok.Margin = new System.Windows.Forms.Padding(8, 4, 8, 4);
+            this._ok.MinimumSize = new System.Drawing.Size(115, 30);
             this._ok.Name = "_ok";
-            this._ok.Size = new System.Drawing.Size(86, 24);
-            this._ok.TabIndex = 1;
+            this._ok.Size = new System.Drawing.Size(115, 30);
+            this._ok.TabIndex = 3;
```

Comment:
Do you have a screen shot you can put in the conversation of how the layout appears with these changes?